wpeverest/everest-forms
# Enhancement - Required field validation(global setting, Individual setting)

### All Submissions:


### Changes proposed in this Pull Request:
In this the users has the flexibility to choose the required field message from global setting or individual setting.

### How to test the changes in this Pull Request:

1. Create a form.
2. Verify required option is working or not
3. and also test in old form it is working or not.

### Types of changes:




### Other information:




### Changelog entry

> Enhancement - Required field validation(global setting, Individual setting)
